include/rocksdb/options.h

```diff
@@ -317,6 +317,13 @@ struct ColumnFamilyOptions : public AdvancedColumnFamilyOptions {
   // Default: nullptr
   std::shared_ptr<SstPartitionerFactory> sst_partitioner_factory = nullptr;
 
+  // Column family based write buffer manager, if this is set, this column
+  // facmily will not report memtable memory usage to the write buffer manager
+  // in DBImpl.
+  //
+  // Default: null
+  std::shared_ptr<WriteBufferManager> cf_write_buffer_manager = nullptr;
```

Comment:
```suggestion
  std::shared_ptr<WriteBufferManager> write_buffer_manager = nullptr;
```

Reply:
Option inherited from DBOptions and ColumnFamilyOptions, so I use different name for it to avoid trouble.